tests/test_grammar.py

```diff
@@ -221,6 +221,8 @@
     ("ws_inter_res_around_colon", "${test\t  : \tfoo}", "foo"),
     ("ws_inter_res_inside_id", "${te st:foo}", GrammarParseError),
     ("ws_inter_res_inside_args", "${test:f o o}", "f o o"),
+    ("ws_inter_res_namespace", "${ns1 .\t ns2 . test:0}", 0),
+    ("ws_inter_res_no_args", "${test: \t}", []),
```

Comment:
Why are whitespaces inside the function name legal?

Reply:
I thought it made sense because we allow whitespaces in node interpolations (`${foo . bar}`). But I don't mind either way.

Reply:
I think both should be illegal.
Whitespace in interpolation keys is not supported in OmegaConf 2.0:

```python
In [9]:  OmegaConf.create({"x" : {"a": 10, "b": "${x. a}"}}).x.b
...
ConfigKeyError: str interpolation key 'x. a' not found
        full_key: x.b
        reference_type=Any
        object_type=dict
```

This can be addressed in a followup diff to keep things manageable :)

Reply:
I think this dates back from the old Hydra grammar where whitespaces were systematically ignored in the lexer.
But I don't mind removing that -- preferably in a different PR indeed since this also touches non resolvers.

Reply:
Yup, let's clean this up.
Hydra should also not accept it as valid.